### PR TITLE
Add in part one of a metals retrospective blog post

### PR DESCRIPTION
**Pull request by ckipp01**
So I want to get a bit of feedback on this. It's really important for me to make sure that we not only share some of the survey results, but also talk about some of the actions that came out of them.

My idea is to have a 2 part series where in part 1 we talk briefly about some of the results, address a couple things that stood out to me when reviewing the initial results, and talk a bit about some of the stuff that we've already done to address certain things. Or just address certain things that already exist that people may not be aware of. My plan would then be to cover the following in the next one, Part 2.

- Build Tool support - what does this look like moving foward
- BSP Server support - talk a bit about solidifying sbt BSP support and also introducing Mill BSP support
- Other large things we'd like to work on this next year
- A special Scala 3 section (that I'll probably need your help with @tgodzik)

What do you all think? Please also let me know if there are other things/sections that you think would be a good fit here. Some of this is probably pretty bare/rough.

### CODE REVIEW

File: website/blog/2021-02-02-metals-retro-part1.md [hunk 1]
```diff
@@ -0,0 +1,173 @@
...
+throughout the survey. However, I want to make a specific point that since the
+inception of Metals, the early maintainers did a fantastic job of ensuring that
+everything would work in clients that offered the necessary LSP support, and at
+times offered support for the Metals specific LSP extensions that we use. This
+has remained core for us a we continue to provide new features. Pretty much any
+new feature that is added is added in a way that can also be utilized for
+editors that may not have the same level of support as VS Code. Here are a few
+examples:
+
+- Debugging support - Metals actually offeres deubugging support for any client
```
Comment (tgodzik) [inline website/blog/2021-02-02-metals-retro-part1.md:70]:
```suggestion
- Debugging support - Metals actually offers deubugging support for any client
```

File: website/blog/2021-02-02-metals-retro-part1.md [hunk 2]
```diff
@@ -0,0 +1,173 @@
...
+    that can server as a DAP client. So for example using `coc-metals` you can
```
Comment (tgodzik) [inline website/blog/2021-02-02-metals-retro-part1.md:71]:
```suggestion
    that can serve as a DAP client. So for example using `coc-metals` you can
```

**Review by tgodzik — APPROVED** [review]
Looks pretty awesome! Thanks for taking the lead on this!

I can for sure help out with the section about what is indented for Scala 3. I tried to cover it in the contributors psot, but for sure it would be useful to have it also somewhere closer to home.

File: website/blog/2021-02-02-metals-retro-part1.md (hunk 2 above)
Comment (tgodzik) [reply website/blog/2021-02-02-metals-retro-part1.md:71]: Freudian slip? :D

Comment (ckipp01) [reply website/blog/2021-02-02-metals-retro-part1.md:71]: 😆 must have been.

**Comment by ckipp01** [comment]
> Looks pretty awesome! Thanks for taking the lead on this!
> 
> I can for sure help out with the section about what is indented for Scala 3. I tried to cover it in the contributors psot, but for sure it would be useful to have it also somewhere closer to home.

Cool, sounds good. Since I envision that one being a bit more in-depth and longer than this one, I'll probably actually create part 2 in the coming weeks as a Google doc and just share it with you for easier editing/collab.

**Pull request merged**